.github/workflows/test.yaml

```diff
@@ -0,0 +1,16 @@
+on: [push, pull_request]
```

Comment:
`push` should only be for the master branch.

Reply:
Done.